realm/realm-library/src/objectServer/java/io/realm/permissions/PermissionOffer.java

```diff
@@ -81,7 +81,7 @@ public PermissionOffer() {
      * @param url specific url to Realm effected this offer encompasses all Realms manged by the user making the offer.
```

Comment:
It's not about the line 81, it's about the line 79:

Creates a request for an permission offer that last until it is manually revoked. -> Creates a request for an permission offer that lasts until it is manually revoked?